pkgs/data/icons/numix-icon-theme-circle/default.nix

```diff
@@ -11,7 +11,7 @@ stdenv.mkDerivation rec {
     owner = "numixproject";
     repo = package-name;
     rev = version;
-    sha256 = "14ck07j9v0yh8ky191sa3xxi4qh7bbg84i8jijy3kbjcx9s0zl8a";
+    sha256 = "10sn2w9xzmh7arbw0vn516p9nfym1bs8bf7i8zzvz7y09s61lkxh";
```

Comment:
Is this hash correct? I get `1dn8gc2rrwm57v7lhjsr91y1w0vwskxi56xzdwrsnnszcfhf4vmd`.

Reply:
I think is is correct:

```
$ nix-prefetch-url -A numix-icon-theme-circle.src
downloading ‘https://github.com/numixproject/numix-icon-theme-circle/archive/17-12-25.tar.gz’... [1301/0 KiB, 421.7 KiB/s]
unpacking...
path is ‘/nix/store/p7wwnan59nh26b63nlna4v33lw7v3bfy-source’
10sn2w9xzmh7arbw0vn516p9nfym1bs8bf7i8zzvz7y09s61lkxh
```